File: VB .NET 2017/EX11_LSS/NodeInput.vb
Public Class NodeInput
    Dim nodeIdentification As Integer

    Private Sub NodeInput_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub

    Private Sub SendIDButton_Click(sender As Object, e As EventArgs) Handles SendIDButton.Click
        nodeIdentification = Convert.ToInt32(NodeIDTextBox.Text)
        Dispose()
    End Sub

    Public Function NodeID() As Integer
        Return nodeIdentification
    End Function
End Class

File: C# .NET 2017/EX11_LSS/NodeInput.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace EX12_LSS
{
    public partial class NodeInput : Form
    {
        int nodeID;

        public NodeInput()
        {
            InitializeComponent();
            ShowDialog();
        }

        private void NodeInput_Load(object sender, EventArgs e)
        {

        }

        private void SendIDButton_Click(object sender, EventArgs e)
        {
            nodeID = Convert.ToInt32(NodeIDTextBox.Text);
            Dispose();
        }

        public int NodeID
        {
            get { return nodeID; }
        }
    }
}
